### ISSUE TITLE
Theano file lock error

### ISSUE THREAD

**Issue by avivajpeyi** [dfm/tess-atlas#264]
Getting 289/3,394 TOI analyses with a `filelock` error:
```
ilelock._error.Timeout: The file lock '.1662466228.theano/compiledir_Linux-5.3-46_6.0.29-cray_ari_c-x86_64-with-glibc2.26-x86_64-3.9.12-64/.lock' could not be acquired.
```

This issue has been flagged already
https://discourse.pymc.io/t/using-pymc3-and-theano-on-cluster-with-slurm-filelock-error-timeout/10316

**Comment by avivajpeyi** [comment]
Maybe just adding `base_compiledir=/var/tmp/.theano` or smth like this will fix this?

**Comment by avivajpeyi** [comment]
Fixed with 121b4ce47d32b23764e287b38992abacd8c2f3c3
